library/common/config/config.cc

```diff
@@ -267,6 +268,9 @@ R"(
                   base_interval: *dns_fail_base_interval
                   max_interval: *dns_fail_max_interval
                 dns_query_timeout: *dns_query_timeout
+                typed_dns_resolver_config:
+                  name: *dns_resolver_name
```

Comment:
Need a default for this. I think it makes sense to just specify c-ares as the default above (along with `dns_resolver_config`, and iOS can still always override it with the apple dns resolves.

Reply:
@goaway I was wondering if this was the case. I make overrides for it on both the java and objc definitions so I saw no point in putting in a default in the common template given there is no common default.

Reply:
Right now the default (header + config) with no overrides is a valid config. I'd like to keep it that way.

One reason is that we can start to use it in tests, instead of the canned configs some tests use (that have tendency to drift).

Reply:
That's fair. Will update.

Reply:
Thank you!